doc/v2/documentation_data_format.rst

```diff
@@ -162,6 +162,19 @@ PEtab 2.0.0 is a major update of the PEtab format. The main changes are:
   clearer. The ``log10`` transformation has been removed, since this was mostly
   relevant for visualization purposes, and the same effect can be achieved by
   rescaling the parameters of the respective (natural) log-distributions.
+* The ``observableFormula`` field in the :ref:`v2_observables_table` must not
+  contain any observable IDs. This was previously allowed, but it was not
+  well-defined how to deal with placeholder parameters in this case.
```

Comment:
Not true? At least, the spec currently says observable IDs can exist in observable formula but circular dependencies must be avoided. Although I don't really understand this -- typo?

Reply:
Thanks, I forgot to remove that sentence down there. It would work for observables without placeholder parameters, but I am not sure if that feature was important enough to add an exception there.

Reply:
Fine for me to exclude :+1: